package.json

```diff
@@ -8,10 +8,10 @@
     "doc": "jsdoc src/Core/View.js src/Core/Prefab/GlobeView.js src/Core/Layer/Layer.js src/Renderer/ColorLayersOrdering.js src/Renderer/ThreeExtended/GlobeControls.js src/Core/Geographic/Coordinates.js src/Core/Scheduler/Providers/GpxUtils.js src/Core/MainLoop.js -t node_modules/docdash --readme JSDOC.md -c jsdoc-config.json",
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint -- --max-warnings=0 && npm run build && npm run test-examples && npm run test-unit",
-    "test-unit": "cross-env BABEL_DISABLE_CACHE=1 mocha --compilers js:babel-core/register test/*unit_test.js",
```

Comment:
babel-plugin-inline-import's README says this should still be used; and with webpack that babel-inline-import-loader could be used instead: https://www.npmjs.com/package/babel-plugin-inline-import#caveats

Reply:
Oh right, thanks for pointing that out. 

> babel-inline-import-loader could be used instead

From the doc of the loader, I understand that this loader actually needs babel-plugin-inline import (so both should be added). Did I understand well?